### PR TITLE
Kenya parameters - contraceptive matrices

### PR DESCRIPTION
**Pull request by avictorious**
### Brief description

- This PR replaces contraceptive matrix data in kenya.py parameters file with new matrices from Kenya PMA 2019-2020.  
Matrices for annual switching and postpartum switching are updated. 
- It does leave Senegal matrices in place in the Kenya file for now. We may want to test with these as we work through scenarios and calibration.  They are a bit of a pain to add back in once deleted.  They are commented out.
- I also created a new file that processes raw matrix csv data into the format that the parameters files need, placed in data_processing folder
- Added Marita's code for creating the matrices for PMA and DHS in R into the data_processing  folder
- Put csv files for PMA and DHS (from 2014) in kenya folder for ease of use
- Closes issues #32, #33 , and #34 

Rider:
- Adds conversion of CPR from percent to rate in kenya parameters file.  This is needed to project out mCPR correctly in scenarios, though not needed to create CPR trend with switching matrices.  Fixes issue #50 

### Type(s) of change
- [ ] Bugfix
- [x] Refactor
- [ ] New feature
- [ ] Other

### Checklist
- My code follows the [style guide](https://github.com/amath-idm/styleguide)
  - [x] Yes
  - [ ] N/A
- I've commented my code
  - [x] Yes
  - [ ] N/A
- I've incremented the version number
  - [x] Yes
  - [ ] N/A
- I've updated the changelog
  - [x] Yes
  - [ ] N/A
- I've added tests and checked code coverage
  - [x] Yes 
  - [ ] N/A

### CODE REVIEW

File: fpsim/locations/kenya.py [hunk 1]
```diff
@@ -632,7 +635,7 @@ def method_probs():
     for starting or changing methods over the first 6 months postpartum.
 
     Data from Senegal DHS contraceptive calendars, 2017 and 2018 combined
-    '''
```
[the remark this answers is not in the document]
Comment (MObrien-IDM) [reply fpsim/locations/kenya.py:635]: Oh, sorry, I see that you're using this for reference only. Do we need this? Or was this to guide your Kenya notation?

[the remark this answers is not in the document]
Comment (avictorious) [reply fpsim/locations/kenya.py:635]: I am leaving this for now.  It's a bit of a pain to substitute back in and we're so early in the process to make sure Kenya works as expected (in scenarios, etc) I'd like to leave them if it's okay with you.  They can hopefully be removed soon.

[the remark this answers is not in the document]
Comment (MObrien-IDM) [reply fpsim/locations/kenya.py:635]: Fine with me - let's add in your comment from the PR info to the notation here.

**Review by MObrien-IDM — APPROVED** [review]

File: fpsim/locations/kenya.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (avictorious) [reply fpsim/locations/kenya.py:635]: Added in documentation and will merge

**Pull request merged**